### PR TITLE
Bug 1525882 , Updated the 3.7 RNs: Enabling Static IPs for External  Project Traffic as Technology Preview

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1525882

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@danwinship PTAL. Looks like this was not clearly labeled as Tech Preview for OCP 3.7 in our release notes, so this fixes that.

File: release_notes/ocp_3_7_release_notes.adoc [hunk 1]
```diff
@@ -755,15 +755,24 @@ $ curl --head https://$route -k
...
-All outgoing external connections from a project will share a single
-fixed-source IP address and will send all traffic via that IP so that external
-firewalls can recognize the application associated with a packet.
+As a cluster administrator, you can assign specific, static IP addresses to
+projects, so that traffic is externally easily recognizable. This is different
+from the default egress router, which is used to send traffic to specific
+destinations.
+
+Recognizable IP traffic increases cluster security by ensuring the origins
+visible. Once enabled, all outgoing external connections from the specified
```
Comment (danwinship) [inline release_notes/ocp_3_7_release_notes.adoc]: Is that suposed to be "by ensuring the origin *is* visible"? Or did even more text accidentally get deleted there?

Comment (danwinship) [reply release_notes/ocp_3_7_release_notes.adoc]: Oh, apparently that was just copied from managing_networking.adoc. Well, same fix is needed there.

**Comment by ahardin-rh** [comment]
@danwinship Good catch. Thanks. This is updated.

**Pull request merged**